### PR TITLE
Help warning on run/exec -u should be on sandbox only

### LINKED ISSUE #2557: run/exec -u option help should warn that only runs with sandbox
## Version of Singularity:

current master

### Expected behavior
```
 -u, --userns                 run container in a new user namespace,
                               allowing Singularity to run completely
                               unprivileged on recent kernels. This may
                               not support every feature of Singularity.
                               (Sandbox image only)
```

### Actual behavior

```
 -u, --userns                 run container in a new user namespace,
                               allowing Singularity to run completely
                               unprivileged on recent kernels. This may
                               not support every feature of Singularity.
```

### Steps to reproduce behavior

```
singularity run/exec --help
```

### PR DESCRIPTION
**Pull request by jchavezlavalle**
**Description of the Pull Request (PR):**

Warning on run and exec should be on sandbox only.

**This fixes or addresses the following GitHub issues:**

- Fixes #2557 

**Before submitting a PR, make sure you have done the following:**

- Read the [Guidelines for Contributing](../CONTRIBUTING.md), and this PR conforms to the stated requirements.
- Added changes to the [CHANGELOG](../CHANGELOG.md) if necessary according to the [Contribution Guidelines](../CONTRIBUTING.md)
- Added tests to validate this PR and tested this PR locally with a `make testall`
- Based this PR against the appropriate branch according to the [Contribution Guidelines](../CONTRIBUTING.md)
- Added myself as a contributor to the [Contributors File](../CONTRIBUTORS.md)

Attn: @singularity-maintainers

### CODE REVIEW

File: CONTRIBUTORS.md
Comment (ArangoGutierrez) [inline CONTRIBUTORS.md:41]: ha!

**Review by ArangoGutierrez — APPROVED** [review]
LGTM
since this is a small behavior change from 2.x, I'll wait for @GodloveD for approval

**Review by GodloveD — APPROVED** [review]
lgtm!

File: internal/app/singularity/action_flags.go
Comment (DrDaveD) [inline internal/app/singularity/action_flags.go:245]: I think the new parenthetical addition is a bit confusing because the previous sentence is in the negative; it isn't completely clear if the parenthetical should also be considered in the negative; does it mean Sandbox image is the only thing *not* supported?  I suggest instead changing the last sentence to "This disables some features of Singularity, for example it only works with sandbox images."

Comment (GodloveD) [reply internal/app/singularity/action_flags.go:245]: Sounds good to me. Want to make the PR @DrDaveD?

Comment (DrDaveD) [reply internal/app/singularity/action_flags.go:245]: #2592

**Pull request merged**
